ubuntu.18.04/Dockerfile

```diff
@@ -6,10 +6,10 @@ ARG Octopus_Client_Version=8.4.0
 ARG Azure_Cli_Version=2.4.0\*
 ARG Azure_Powershell_Version=2.2.0
 ARG Helm_Version=v3.0.2
-ARG Node_Version=12.16.3\*
+ARG Node_Version=12.18.2\*
 ARG Kubectl_Version=1.11.1-00
 ARG Terraform_Version=0.12.24
-ARG Eks_Cli_Version=0.18.0
+ARG Eks_Cli_Version=0.23.0

```

Comment:
Do these changes cause the image version to be bumped?

Reply:
The build scripts build a gitversion tag, so the merge will generate a new minor release.